py-polars/tests/unit/io/test_csv.py

```diff
@@ -1060,25 +1060,27 @@ def test_csv_single_categorical_null() -> None:
 
 
 def test_csv_quoted_missing() -> None:
-    csv = '''"col1"|"col2"|"col3"|"col4"
-"0"|"Free text with a line
-break"|"123"|"456"
-"1"|"Free text without a linebreak"|""|"789"
-"0"|"Free text with 
-two 
-linebreaks"|"101112"|"131415"'''  # noqa: W291
-    assert pl.read_csv(csv.encode(), sep="|", dtypes={"col3": pl.Int32}).to_dict(
-        False
-    ) == {
-        "col1": [0, 1, 0],
-        "col2": [
-            "Free text with a line\nbreak",
-            "Free text without a linebreak",
-            "Free text with \ntwo \nlinebreaks",
-        ],
-        "col3": [123, None, 101112],
-        "col4": [456, 789, 131415],
-    }
+    csv = (
+        '"col1"|"col2"|"col3"|"col4"\n'
+        '"0"|"Free text with a line\nbreak"|"123"|"456"\n'
+        '"1"|"Free text without a linebreak"|""|"789"\n'
+        '"0"|"Free text with \ntwo \nlinebreaks"|"101112"|"131415"'
+    )
```

Comment:
This was a multiline string with trailing spaces, and since my IDE strips trailing whitespace I rewrote it like this.